### PR TITLE
update outdated docs on console logging configuration

### LINKED ISSUE #6172: logging error with 1.2.0 version
## 🐛 Bug

if I use logger named "lightning", logger print log twice.
It happens with pytorch-lightning  1.2.0 version and doesn't happend with 1.1.8 version. 
This is reproduced easily like below.

Error Example 
```python
Type "help", "copyright", "credits" or "license" for more information.
>>> import pytorch_lightning as pl
>>> import logging
>>> logger = logging.getLogger("lightning")
>>> logger.info("test")
test
>>> logging.basicConfig(level=logging.INFO)
>>> logger.info("test")
test
INFO:lightning:test
>>> pl.__version__
'1.2.0'
```
```python
>>> import pytorch_lightning as pl
>>> import logging
>>> logger = logging.getLogger("lightning")
>>> logger.info("test")
>>> logging.basicConfig(level=logging.INFO)
>>> logger.info("test")
INFO:lightning:test
>>> pl.__version__
'1.1.8'
```

## Please reproduce using the BoringModel

Don't need model.

### To Reproduce

Refer above codes 1.2.0 version

### Expected behavior

Refer above codes 1.1.8 version

### Environment

 - PyTorch Version (e.g., 1.0): 1.2.0
 - OS (e.g., Linux): Ubuntu 20.04.1 LTS
 - How you installed PyTorch (`conda`, `pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.7,9
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

Fixes #6172

These docs are outdated, the logger is attached to the package name and submodules name. 

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [ ] Is this pull request ready for review? (if not, please submit in draft mode)
 - [ ] Check that all items from **Before submitting** are resolved
 - [ ] Make sure the title is self-explanatory and the description concisely explains the PR
 - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/extensions/logging.rst [hunk 1]
```diff
@@ -265,7 +265,7 @@ to see fewer messages like so:
 .. code-block:: python
 
     import logging
-    logging.getLogger("lightning").setLevel(logging.ERROR)
+    logging.getLogger("pytorch_lightning").setLevel(logging.ERROR)
```
Comment (SkafteNicki) [inline docs/source/extensions/logging.rst:268]: should this not be consistent with what we have in `__init__`?
https://github.com/PyTorchLightning/pytorch-lightning/blob/ed67490d93e3f2bd46ba50cc53207492da0df73d/pytorch_lightning/__init__.py#L41

Comment (awaelchli) [reply docs/source/extensions/logging.rst:268]: I am heavily confused right now
All changes from this PR here #5509 
seem to have disappeared

Comment (SkafteNicki) [reply docs/source/extensions/logging.rst:268]: must be gone in the process when we released v1.2?

Comment (awaelchli) [reply docs/source/extensions/logging.rst:268]: ok, I am sad
Then it's better I restore the old branch and try to resend the PR
Thanks for helping me notice that @SkafteNicki

**Pull request closed without merging**